swagger/v2/move.yaml

```diff
@@ -146,3 +146,6 @@ Move:
         timeline_events:
           $ref: "../v1/event_reference.yaml#/EventReference"
           description: Aliased events about this record and its nested relationships
+        journeys:
+          $ref: "../v1/journey_reference.yaml#/JourneyReference"
```

Comment:
I assume there's no v2 file?

Reply:
we only have v2 files for the different documentation between v1 and v2. If there is only one it currently lives in v1